src/client/components/board/board.css

```diff
@@ -14,4 +14,12 @@
   left: 5px;
   bottom: 5px;
   background-color: white;
+}
+
+.player-wrap {
+  position: relative;
+}
+
+.player-wrap:hover {
+  z-index: 1;
```

Comment:
I like this idea!
Before, the played card covered the deck on screen with a small height.
Now, the card deck "gets focus" on hovering and is still clickable.

However, this introduces the following behavior (which I find rather annoying): Also on large screens, the clock disappears when you hover the card deck. Maybe choose the z-indices in such way that the clock is always visible? What to you think?

Reply:
You are right, I completely forgot about the clock. I changed the clock's z-index so it's now always on top.
Just as a note for the future: If we're going to use z indices more often, we should think about using more semantic values when it's clear what the different z layers are. For now I just chose the simplest values possible since these are the only z-index values in the project so far.